src/telemetry/telemetry.c

```diff
@@ -89,25 +64,17 @@ telemetry_parse_version(const char *json, const long installed_version[3], Versi
 	}
 
 	/*
-	 * Now parse the version string. We expect format to be XX.XX.XX, and if
-	 * not, we error out
+	 * Now parse the version string. We expect format to be
+	 * XX.XX.XX-<prerelease_tag>, and if not, we error out
 	 */
-	if (!parse_version_string(result->versionstr, result->version))
+	if (!version_parse(result->versionstr, &result->vinfo))
```

Comment:
Should we add the unparsed version to the error hint when `version_parse` fails?